### PR TITLE
Add a JIRA Realm

### PR DESCRIPTION
**Pull request by Kegsay**

### CODE REVIEW

File: src/github.com/matrix-org/go-neb/realms/jira/jira.go [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+func loadPrivateKey(privKeyPEM string) (*rsa.PrivateKey, error) {
+	// Decode PEM to grab the private key type
+	block, _ := pem.Decode([]byte(privKeyPEM))
+	if block == nil {
+		return nil, errors.New("No PEM formatted block found")
+	}
+
+	// TODO: Handle passwords on private keys.
+	// decBytes, err = x509.DecryptPEMBlock(block, []byte{}) // no pass
```
Comment (NegativeMjark) [inline src/github.com/matrix-org/go-neb/realms/jira/jira.go]: Do we need this TODO?

**Comment by NegativeMjark** [comment]
Other than the possibly spurious TODO. LGTM.

File: src/github.com/matrix-org/go-neb/realms/jira/jira.go (hunk 1 above)
Comment (Kegsay) [inline src/github.com/matrix-org/go-neb/realms/jira/jira.go]: Given the key is sent remotely, probably not. Will remove.

**Comment by NegativeMjark** [comment]
LGTM

**Pull request merged**